install

```diff
@@ -262,7 +262,7 @@ if should_install_command_line_tools?
   # This temporary file prompts the 'softwareupdate' utility to list the Command Line Tools
   clt_placeholder = "/tmp/.com.apple.dt.CommandLineTools.installondemand.in-progress"
   sudo "/usr/bin/touch", clt_placeholder
-  clt_label = `softwareupdate -l | grep -B 1 -E "Command Line (Developer|Tools)" | awk -F"*" '/^ +\\*/ {print $2}' | sed 's/^ *//' | head -n1`.chomp
+  clt_label = `/usr/sbin/softwareupdate -l | /usr/bin/sed -E '/^.*\*\ *(Command Line (Developer|Tools).*)\ *$/!d;$!d;s//\1/'`.chomp
```

Comment:
No need to hardcode these paths. Can you explain a bit what the regex is matching?